dashboard/basicPreferences.source.ts

```diff
@@ -106,3 +109,54 @@ solid:PowerUser a rdfs:Class;
 
 export default basicPreferencesPane
 // ends
+
+function addDeletionLinks (container: HTMLElement, kb: IndexedFormula, profile: NamedNode): void {
+  const podServerNodes = kb.each(profile, ns.space('storage'), null, profile.doc())
+  const podServers = podServerNodes.map(node => node.value)
+  podServers.forEach(async (server) => {
+    const deletionLink = await generateDeletionLink(server)
+    if (deletionLink) {
+      container.appendChild(deletionLink)
+    }
+  })
+}
+
+async function generateDeletionLink (podServer: string): HTMLElement {
+  const link = document.createElement('a')
+  link.textContent = `Delete your account at ${podServer}`
+  const deletionUrl = await getDeletionUrlForServer(podServer)
+  if (typeof deletionUrl !== 'string') {
+    return null
+  }
+  link.href = deletionUrl
+  return link
+}
+
+/**
+ * Hacky way to get the deletion link to a Pod
+ *
+ * This function infers the deletion link by assuming the URL structure of Node Solid server.
+ * In the future, Solid will hopefully provide a standardised way of discovering the deletion link:
+ * https://github.com/solid/data-interoperability-panel/issues/18
+ *
+ * @param server Pod server containing the user's account.
+ * @returns URL of the page that Node Solid Server would offer to delete the account, or null if
+ *          the URLs we tried give invalid responses.
+ */
+async function getDeletionUrlForServer (server: string): Promise<string | null> {
```

Comment:
An alternative version could be to simple give a list of URLs to check and use `Promise.allSettled`, instead of doing one by one - but this is a hack in any case, and I don't have a problem with this solution (which could of course save one request).

Reply:
The disadvantage of that is that it will make all HTTP requests at the same time, whereas with the current code it will not make the second request if the first one resolves. (Though it'd probably be good to first try it for the multi-user setup, which is more likely - I'll change that.)

An array of URLs could work, but that's optimising for being able to add more URLs later - I'd hope that that would not be necessary, thanks to https://github.com/solid/data-interoperability-panel/issues/18.